crates/yttrium/src/chain_abstraction/api/fungible_price.rs

```diff
@@ -23,6 +23,8 @@ pub struct PriceRequestBody {
     pub project_id: ProjectId,
     pub currency: Currency,
     pub addresses: HashSet<String>,
+    #[serde(default)]
+    pub sdk_versions: String,
```

Comment:
Need to consider format here. AppKit uses `st` and `sv`: https://github.com/reown-com/appkit/pull/3764/files#diff-ab264fc46b4841a6d655b1222419dbfdf0ffe800f6a9ae24ae7b1c32ede22c6bR140